filebeat/module/postgresql/log/test/postgresql-9.6-debian-with-slowlog.log-expected.json

```diff
@@ -1,6 +1,6 @@
 [
     {
-        "@timestamp": "2017-07-31T13:36:42.585Z",
+        "@timestamp": "2017-07-31T11:36:42.585Z",
```

Comment:
This means we had previously a bug in here?

Reply:
I'd say so, yes. It was not using the timezone for date parsing before.

Reply:
Ok, labeled PR as a bug 👍